### PR TITLE
commit

### PR DESCRIPTION
**Pull request by vaibhav-d-simformsolutions**
practice with validator package and chalk package

### CODE REVIEW

**Comment by sumitsimform** [comment]
@vaibhav-d-simformsolutions  : please create .gitignore  and readme file 
and then don't commit the node moudules folder.

File: node_modules/ansi-styles/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
+declare type CSSColor =
+	| 'aliceblue'
+	| 'antiquewhite'
```
Comment (sumitsimform) [inline node_modules/ansi-styles/index.d.ts:3]: No need to commit the node modules folder.

**Review by sumitsimform — CHANGES_REQUESTED** [review]

File: node_modules/ansi-styles/index.d.ts (hunk 1 above)
Comment (vaibhav-d-simformsolutions) [reply node_modules/ansi-styles/index.d.ts:3]: do have i to commit only js files?

Comment (sumitsimform) [reply node_modules/ansi-styles/index.d.ts:3]: remove node modules folder only from the commit.

Comment (vaibhav-d-simformsolutions) [reply node_modules/ansi-styles/index.d.ts:3]: or json files?

**Pull request merged**